### PR TITLE
PM-384: Filters don't work

### PR DESCRIPTION
**Pull request by mmvsha**

### CODE REVIEW

File: src/selectors/market.js [hunk 1]
```diff
@@ -76,9 +76,11 @@ export const filterMarkets = state => (opts) => {
 export const sortMarkets = (markets = [], orderBy = null) => {
   switch (orderBy) {
   case 'RESOLUTION_DATE_ASC':
-    return markets.sort((a, b) => a.eventDescription.resolutionDate > b.eventDescription.resolutionDate)
+    return markets.sort((a, b) =>
+      new Date(a.eventDescription.resolutionDate) - new Date(b.eventDescription.resolutionDate))
```
Comment (apanizo) [inline src/selectors/market.js]: What is ```resolutionDate```?

Comment (mmvsha) [reply src/selectors/market.js]: @apanizo Approximate date when market is likely going to be resolved.

Comment (mmvsha) [reply src/selectors/market.js]: @apanizo Its an approximate date when market is likely going to be resolved. It is shown in market details as "Resolution Date". I think it's clear

Comment (apanizo) [reply src/selectors/market.js]: I mean, what kind of JS variable? Date object? ts in milliseconds? object?

Comment (mmvsha) [reply src/selectors/market.js]: @apanizo its a string with date

**Pull request merged**